backend/game_server/lib/helpers/vector.ex

```diff
@@ -0,0 +1,41 @@
+defmodule BernacleServer.Helpers.RaiseNotVector do 
+	defexception message: "One of arguments is not Vector" 
+end
+
+defmodule BernacleServer.Helpers.Vector do
+	alias :math, as: Math
+	alias BernacleServer.Helpers.{Vector, RaiseNotVector}
+
+	defstruct x: 0.0, y: 0.0
+
+	def distance(v1 = %Vector{}, v2 = %Vector{}) do 
+		Math.sqrt(Math.pow(v1.x - v2.x, 2) + Math.pow(v1.y - v2.y, 2))
+	end
+	
+	def sum(v1 = %Vector{}, v2 = %Vector{}) do
+		%Vector{x: v1.x+v2.x, y: v1.y+v2.y}
```

Comment:
please, be consistent with spaces in expressions.
I believe better way is to use space between any operators and it's arguments.
`v1.x + v2.x`